template/index.html

```diff
@@ -10,54 +10,45 @@
   <link href="css/style.css" rel="stylesheet" media="screen, print">
   <link href="img/favicon.ico" rel="icon" type="image/x-icon">
   <script src="vendor/polyfill.js"></script>
+  <script lang="javascript">
+    window.onload = function() {
```

Comment:
this is not needed, apidoc takes care of scrolling the menu to current item by itself